<?xml version="1.0" encoding="utf-8"?>
<test-run id="2" testcasecount="1" result="Failed(Child)" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0" engine-version="3.5.0.0" clr-version="4.0.30319.42000" start-time="2019-04-11 18:39:59Z" end-time="2019-04-11 18:42:59Z" duration="180.0706131">
  <test-suite type="TestSuite" id="1000" name="New Unity Project" fullname="New Unity Project" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-04-11 18:39:59Z" end-time="2019-04-11 18:42:59Z" duration="180.070613" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0">
    <properties />
    <failure>
      <message><![CDATA[One or more child tests had errors]]></message>
    </failure>
    <test-suite type="Assembly" id="1005" name="Tests.dll" fullname="C:/Users/jackpr/Git/CertTestRunnerScratch/Library/ScriptAssemblies/Tests.dll" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-04-11 18:39:59Z" end-time="2019-04-11 18:42:59Z" duration="180.049885" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0">
      <properties>
        <property name="_PID" value="1212" />
        <property name="_APPDOMAIN" value="Unity Child Domain" />
      </properties>
      <failure>
        <message><![CDATA[One or more child tests had errors]]></message>
      </failure>
      <test-suite type="TestSuite" id="1006" name="Tests" fullname="Tests" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-04-11 18:39:59Z" end-time="2019-04-11 18:42:59Z" duration="180.043682" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0">
        <properties />
        <failure>
          <message><![CDATA[One or more child tests had errors]]></message>
        </failure>
        <test-suite type="TestFixture" id="1002" name="NewTestScript" fullname="Tests.NewTestScript" classname="Tests.NewTestScript" runstate="Runnable" testcasecount="2" result="Failed" site="Child" start-time="2019-04-11 18:39:59Z" end-time="2019-04-11 18:42:59Z" duration="180.035382" total="1" passed="0" failed="1" inconclusive="0" skipped="0" asserts="0">
          <properties />
          <failure>
            <message><![CDATA[One or more child tests had errors]]></message>
          </failure>
          <test-case id="1004" name="TestScene2" fullname="Tests.NewTestScript.TestScene2" methodname="TestScene2" classname="Tests.NewTestScript" runstate="Runnable" seed="1837648814" result="Failed" start-time="2019-04-11 18:39:59Z" end-time="2019-04-11 18:42:59Z" duration="180.012465" asserts="0">
            <properties>
              <property name="_JOINTYPE" value="UnityCombinatorial" />
              <property name="Description" value="This test should load the scene, print a debug statement, and succeed" />
            </properties>
            <failure>
              <message><![CDATA[Test exceeded Timeout value of 180000ms]]></message>
            </failure>
            <output><![CDATA[
General Status: Passed 
Detailed Status: scene was loaded, you are now getting test status.  The setup works.
]]></output>
          </test-case>
        </test-suite>
      </test-suite>
    </test-suite>
  </test-suite>
</test-run>